Kitodo-API/src/main/java/org/kitodo/api/dataformat/Workpiece.java

```diff
@@ -37,11 +36,10 @@
     private String id;
 
     /**
-     * The media units that belong to this workpiece. The order of this
-     * collection is meaningful, but only describes the order in which the media
-     * units are displayed on the workstation of the compiler.
+     * The media unit that belongs to this workpiece. The media unit can have
+     * children, such as a bound book that can have pages.
      */
-    private List<MediaUnit> mediaUnits = new LinkedList<>();
+    private MediaUnit mediaUnit = new MediaUnit();
```

Comment:
I am not sure I understand the purpose of this change. Doesn't this mean the workpiece / root element of the physical structure can always only have one child element / subordinate unit, which than in turn can have multiple children? If that's the case, I think you should keep the existing list of `MediaUnit` objects as a field for `Workpiece`, since we do want multiple direct children beneath the root element.

Reply:
This allows access to the BoundBook. And e.g. its metadata.